helm/g2p-sandbox-fynarfin-SIT/config/application-tenants.properties

```diff
@@ -1,6 +1,8 @@
 bpmns.tenants[0].id= lion
-bpmns.tenants[0].flows.payment-transfer= mock_payment_transfer-{dfspid}
+bpmns.tenants[0].flows.payment-transfer= minimal_mock_fund_transfer-{dfspid}
 bpmns.tenants[1].id= rhino
-bpmns.tenants[1].flows.payment-transfer= minimal_mock_fund_transfer-{dfspid}
+bpmns.tenants[1].flows.payment-transfer= minimal_mock_fund_transfer_account_lookup-{dfspid}
+bpmns.tenants[1].flows.outbound-transfer-request= minimal_mock_transfer_request-{dfspid}
```

Comment:
@Anover000 : Where is the bpmn file for minimal_mock_transfer_request-{dfspid} in version control?

Reply:
missed while getting prs merged for this task. Have raised it now - https://github.com/openMF/ph-ee-env-labs/pull/183